cabby/rvs/generate_rvs.py

```diff
@@ -58,45 +59,44 @@ def main(argv):
   bearing = util.get_bearing(start, supplied_goal)
   print(f'Distance {path_distance} km | Bearing {bearing}')
   
-  # Get all Wikidata entities in Manhattan region and create dictionary from
-  # each entity's QID to the entity representation.
-  entities = {}
-  for result in query.get_geofenced_wikidata_items('Manhattan'):
-    entity = item.WikidataEntity.from_sparql_result(result)
-    entities[entity.qid] = entity
+  # Get all OSM in Manhattan region.
+  map = map_structure.Map("Manhattan", 18, "/mnt/hackney/data/cabby/poi/v1/")
```

Comment:
Two things:
 - shouldn't the region be a command line option?
 - shouldn't reference specific on disk location

Reply:
This is ofcourse not incorrect. I meant to changed it and forgot. 
 
Changed and added:
bazel-bin/cabby/rvs/generate_rvs \
  --start_lat 44.49582 --start_lon 11.33697 \
  --goal_lat 44.49268 --goal_lon 11.34365 \
	--region Bologna \
	--min_s2_level 18 \
	--directory "./cabby/geo/map_processing/poiTestData/" \

It runs and the result instruction makes sense.

Reply:
Great!